kuma/wiki/helpers.py

```diff
@@ -355,3 +355,12 @@ def wiki_url(context, path):
     # finally cache the reversed document URL for a bit
     memcache.set(u'wiki_url:%s:%s' % (locale, path), url, 60 * 5)
     return url
+
+
+@register.function
```

Comment:
Done! The test still runs fine with it called as a function, is it okay to leave the test?